docs/source/kale.embed.rst

```diff
@@ -14,6 +14,14 @@ kale.embed.attention\_cnn module
    :undoc-members:
    :show-inheritance:
 
+kale.embed.deepdta module
+--------------------------------
+
+.. automodule:: kale.embed.deepdta
+   :members:
+   :undoc-members:
+   :show-inheritance: bysource
```

Comment:
Are there some references for these options? I found you indicated `member-order` as `bysource` for `gcn` and `gripnet`. I am not familiar and just curious. If these are good practices, we should have them specified for all.